sustainability-best-practices/telemetry/2024-02-telemetry.md

```diff
@@ -0,0 +1,44 @@
+# Telemetry Status February 2024
+
+Last year, around this time, we released our [first telemetry report](https://umbraco.com/blog/how-umbraco-cms-is-used-telemetry-status-february-2023/). However, most of all, it was a disclaimer because we only had few sites, that opted in to detailed telemetry. We asked for your help and hoped to also count your project next year - which is now this year. And you did opt in! The number of responses has increased by 7000%, and there is now no reason for a disclaimer. #H5YR
```

Comment:
```suggestion
Last year, around this time, we released our [first telemetry report](https://umbraco.com/blog/how-umbraco-cms-is-used-telemetry-status-february-2023/). However, most of all, it was a disclaimer because we only had a few sites, that opted into detailed telemetry. We asked for your help and hoped to also count your project next year - which is now this year. And you did opt in! The number of responses has increased by 7000%, and there is now no reason for a disclaimer. #H5YR
```